SQL/New_patches/2018_07_23_battery_manager_permissions.sql

```diff
@@ -0,0 +1,39 @@
+-- Add Battery Manager to LorisMenu
+INSERT INTO LorisMenu (Parent, Label, Link, Visible, OrderNumber)
+       VALUES (
+           (SELECT ID FROM (SELECT ID FROM LorisMenu WHERE Label = 'Admin') as lm),
+           'Battery Manager',
+           'battery_manager/',
+           null,
+           7
+       );
+
+-- Add view permission for battery manager
+INSERT INTO permissions (code, description, categoryID)
+       VALUES (
+           'battery_manager_view',
+           'View Battery Manager',
+           (SELECT ID FROM permissions_category WHERE Description = 'Permission')
+       );
+
+-- Add edit permission for battery manager
+INSERT INTO permissions (code, description, categoryID)
+       VALUES (
+           'battery_manager_edit',
+           'Add, activate, and deactivate entries in Test Battery',
+           (SELECT ID FROM permissions_category WHERE Description = 'Permission')
+       );
+
+-- Give view permission to admin
+INSERT INTO user_perm_rel (userID, permID)
+       VALUES (
+           (SELECT ID FROM users WHERE UserID = 'admin'),
+           (SELECT permID FROM permissions WHERE code = 'battery_manager_view')
+       );
+
```

Comment:
@HenriRabalais I don't have a a UserID named 'admin' in my database. So the above SQL statement failed for me.
If you are intending to give permission to super user; then I think most reliable way is to get the UserID having super user permission.
Usually there will be only one super user; but in some projects it can be more than one. 

@ridz1208 @driusan correct me if I am wrong.

Reply:
That makes sense to me.

Reply:
Okay, sounds good.

But what do I do in the case where there's more than one super user?

Reply:
@driusan @sruthymathew123 That would not really work for me... we give superuser to quite a few people given that its necessary for some modules like data-release. I really dont wanna automatically give them access to the battery manager... I can't even imagine the damage they will do!! (i know they can acces it anyways because superuser always returns true BUT we did remove the need for superuser in the next version of loris so I dont want this permission to accidentally be given to them even after the loose superuser)

And btw.. the admin is in the loris schema (https://github.com/aces/Loris/blob/minor/SQL/0000-00-00-schema.sql#L242) It's fair game to use it in PRs no ?

Reply:
I changed the insert statement to one suggested by @ridz1208. Let me know if it works!